CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,217 @@
+Quansight takes reports of misconduct very seriously and is committed to
+preserving and maintaining the welcoming nature of our community and our company
+culture. You are encouraged to be an active bystander especially if you notice a
+dangerous situation or someone in distress.
+
+Should you observe a breach of the Code of Conduct—even if the breach seems
+inconsequential and/or unintentional—you may choose to point the person(s)
+directly to this Code of Conduct. This can happen in private or
+not—whichever is most appropriate. The goal is to treat all parties with dignity
+and respect; ideally, the offending person(s) should have the opportunity to
+correct the behavior and make amends. If you would prefer not to address the
+person(s) directly, please discuss the issue in confidence with a member of the
+COC committee. You can alternatively seek confidential advice from one of the
+Quansight Labs Co-Directors.
+
+In the event of an egregious incident, please report the matter formally—as
+described below—in a timely manner to the entire COC committee (`coc@quansight.com`),
+to any individual member of the COC committee, or to one of the Quansight
+Labs Co-Directors (listed above).
+
+**All reports will be kept confidential.**
+
+Incident reports will be reviewed and investigated by the COC committee. Possible
+outcomes of the process are detailed below.
+
+### a. How to Submit an Incident Report
+
+In the context of this Code of Conduct, a "report" means a written notification
+(as described below) of any Code of Conduct incident sent to any member of the COC
+committee or a Quansight Labs Co-Director.
+
+Quansight is committed to promptly addressing any incident reports in a timely manner.
+
+### b. Conflicts of Interest
+
+In the event of a conflict of interest with any COC committee member, the relevant member will recuse themselves from handling the report. Involved parties are never part of the review process.
+
+If you are concerned about submitting a report that will reach any specific individual, please contact one of the COC committee members or Quansight Labs Co-Directors directly.
+
+### c. What to Include in a Report
+
+Our ability to address any Code of Conduct incident in a timely and effective manner is critically dependent on the amount & the precision of all information provided. Please include as much of the following as possible in an incident report:
+
++ Your contact information (including your preferred medium of contact).
+    + If you wish to remain anonymous, your personal contact details will not be shared beyond
+      the person receiving the initial report.
++ Approximate time & location of the incident (be as specific as possible).
++ Identifying information of all individuals actively involved in the COC incident.
+    + This may include names, nicknames/pseudonyms, usernames/screen-names, physical
+  descriptions, etc.
++ Witnesses (if any)
```

Comment:
Maybe change to "(if any, in case of an in-person incident)". Or remove. It looks kind of odd now; this applies for a conference, but we're not planning any in-person events.

Reply:
Let's keep it with your suggested caveat... just in case an in-person interaction does happen.